Sources/Email/Emailable.swift

```diff
@@ -0,0 +1,19 @@
+public protocol Emailable {
```

Comment:
Maybe `EmailUser` or something would be a better name here?

Or, perhaps using the `...Representable` pattern. 

```swift
protocol EmailAddressRepresentable {
    var emailAddress: EmailAddress { get }
}
```